feature/experimental/tunnel/otg_tests/tunnel_interface_based_resize_test/tunnel_interface_based_resize_test.go

```diff
@@ -0,0 +1,1572 @@
+		r0Lo0Ut0Ipv4Add: "198.18.70.1",
+		r0Lo0Ut1Ipv4Add: "198.18.71.1",
+		r0Lo0Ut2Ipv4Add: "198.18.72.1",
+		r0Lo0Ut3Ipv4Add: "198.18.73.1",
+		ipv4Mask:        24,
+		ipv4FullMask:    32,
+		r1Intf5Ipv4Add:  "198.18.30.2",
+		r1Intf6Ipv4Add:  "198.18.31.2",
+		r1Intf3Ipv4Add:  "198.18.20.2",
+		r1Intf4Ipv4Add:  "198.18.21.2",
+		r1Fti0Ipv4Add:   "198.18.90.2",
+		r1Fti1Ipv4Add:   "198.18.91.2",
+		r1Fti2Ipv4Add:   "198.18.92.2",
+		r1Fti3Ipv4Add:   "198.18.93.2",
+		r1Fti4Ipv4Add:   "198.18.94.2",
+		r1Fti5Ipv4Add:   "198.18.95.2",
+		r1Fti6Ipv4Add:   "198.18.96.2",
+		r1Fti7Ipv4Add:   "198.18.97.2",
+		r1Lo0Ut0Ipv4Add: "198.18.80.1",
+		r1Lo0Ut1Ipv4Add: "198.18.81.1",
+		r1Lo0Ut2Ipv4Add: "198.18.82.1",
+		r1Lo0Ut3Ipv4Add: "198.18.83.1",
+		rtIntf1Ipv6Add:  "2001:DB8:1:198:18:10::2",
+		rtIntf2Ipv6Add:  "2001:DB8:1:198:18:11::2",
+		rtIntf5Ipv6Add:  "2001:DB8:1:198:18:30::1",
+		rtIntf6Ipv6Add:  "2001:DB8:1:198:18:31::1",
+		r0Intf1Ipv6Add:  "2001:DB8:1:198:18:10::1",
+		r0Intf2Ipv6Add:  "2001:DB8:1:198:18:11::1",
+		r0Intf3Ipv6Add:  "2001:DB8:1:198:18:20::1",
+		r0Intf4Ipv6Add:  "2001:DB8:1:198:18:21::1",
+		r0Fti0Ipv6Add:   "2001:DB8:1:198:18:90::1",
+		r0Fti1Ipv6Add:   "2001:DB8:1:198:18:91::1",
+		r0Fti2Ipv6Add:   "2001:DB8:1:198:18:92::1",
+		r0Fti3Ipv6Add:   "2001:DB8:1:198:18:93::1",
+		r0Fti4Ipv6Add:   "2001:DB8:1:198:18:94::1",
+		r0Fti5Ipv6Add:   "2001:DB8:1:198:18:95::1",
+		r0Fti6Ipv6Add:   "2001:DB8:1:198:18:96::1",
+		r0Fti7Ipv6Add:   "2001:DB8:1:198:18:97::1",
+		r0Lo0Ut0Ipv6Add: "2001:DB8:1:198:18:70::1",
+		r0Lo0Ut1Ipv6Add: "2001:DB8:1:198:18:71::1",
+		r0Lo0Ut2Ipv6Add: "2001:DB8:1:198:18:72::1",
+		r0Lo0Ut3Ipv6Add: "2001:DB8:1:198:18:73::1",
+		ipv6Mask:        120,
+		r1Intf5Ipv6Add:  "2001:DB8:1:198:18:30::2",
+		r1Intf6Ipv6Add:  "2001:DB8:1:198:18:31::2",
+		r1Intf3Ipv6Add:  "2001:DB8:1:198:18:20::2",
+		r1Intf4Ipv6Add:  "2001:DB8:1:198:18:21::2",
+		r1Fti0Ipv6Add:   "2001:DB8:1:198:18:90::2",
+		r1Fti1Ipv6Add:   "2001:DB8:1:198:18:91::2",
+		r1Fti2Ipv6Add:   "2001:DB8:1:198:18:92::2",
+		r1Fti3Ipv6Add:   "2001:DB8:1:198:18:93::2",
+		r1Fti4Ipv6Add:   "2001:DB8:1:198:18:94::2",
+		r1Fti5Ipv6Add:   "2001:DB8:1:198:18:95::2",
+		r1Fti6Ipv6Add:   "2001:DB8:1:198:18:96::2",
+		r1Fti7Ipv6Add:   "2001:DB8:1:198:18:97::2",
+		r1Lo0Ut0Ipv6Add: "2001:DB8:1:198:18:80::1",
+		r1Lo0Ut1Ipv6Add: "2001:DB8:1:198:18:81::1",
+		r1Lo0Ut2Ipv6Add: "2001:DB8:1:198:18:82::1",
+		r1Lo0Ut3Ipv6Add: "2001:DB8:1:198:18:83::1",
+		ipv6FullMask:    128,
+		flow1:           "IPv4-flow1",
+		flow2:           "IPv4-flow2",
+		flow3:           "IPv6-flow3",
+		flow4:           "IPv6-flow4",
+		trafficDuration: 60,
+		trafficRate:     1000,
+    }
+
+	t.Logf("the input variable %+v", p)
+	t.Helper()
+
+	dut1 := ondatra.DUT(t, "dut1")
+	dut1Intf1 := dut1.Port(t, "port1")
+	dut1Intf2 := dut1.Port(t, "port2")
+	dut1Intf3 := dut1.Port(t, "port3")
+	dut1Intf4 := dut1.Port(t, "port4")
+	rt := ondatra.ATE(t, "ate")
+	t.Logf("dut-1 %v", dut1)
```

Comment:
unnecessary to log the dut and the ports, as this is already printed in the reservation information